lib/serializers.rb

```diff
@@ -1,9 +1,11 @@
 module Serializers
-  def save(games, authors, books)
+  def save(games, authors, books, _musicalbums, _genres)
     files = [
       { name: 'games', payload: games },
       { name: 'authors', payload: authors },
-      { name: 'books', payload: books }
+      { name: 'books', payload: books },
+      { name: 'musicalbums', payload: books },
+      { name: 'genres', payload: books }
```

Comment:
Check this please!

Reply:
![image](https://user-images.githubusercontent.com/92180054/182620118-ab219ab7-2ef2-4d7b-9dee-6d07b3c58218.png)
As you can see in the picture above, when I run the app it throws a wrong message and if something is created it is saved as a book in a different file. It could break all the application.

Reply:
![image](https://user-images.githubusercontent.com/92180054/182621223-67fff695-760c-4c78-b25c-0a30ffae1012.png)
The test looks great!!! My suggestion should be to split the context into different statements:
- It should have a published date
- It should be published on spotify
- The same with archived propiety.

Reply:
If you haven't yet implemented these sections, kindly try not to include this yet and leave it for a later activity as it may mess up the currently working program.